src/Microsoft.Tye.Hosting/FuncFinder.cs

```diff
@@ -50,50 +51,62 @@ public Task StartAsync(Application application)
 
         private string? FindFuncForVersion(AzureFunctionRunInfo func)
         {
-            var funcDllPath = FuncDllPath();
+            int version = 4; // Default to the latest version (v4).
+            
+            if (!string.IsNullOrWhiteSpace(func.AzureFunctionsVersion))
+            {
+                var match = Regex.Match(func.AzureFunctionsVersion, @"^[vV](?<number>\d+)$");
```

Comment:
If the regular expression is adjusted to `^[vV]?(?<number>\d+)$`, then Tye supports with and without version prefix.

Reply:
Sounds reasonable; updated.